src/tools/perf/perftest.c

```diff
@@ -80,6 +80,12 @@ test_type_t tests[] = {
     {"ucp_put_bw", UCX_PERF_API_UCP, UCX_PERF_CMD_PUT, UCX_PERF_TEST_TYPE_STREAM_UNI,
      "put bandwidth", "overhead", 32},
 
+     {"ucp_put_multi_bw", UCX_PERF_API_UCP, UCX_PERF_CMD_PUT_MULTI, UCX_PERF_TEST_TYPE_STREAM_UNI,
+     "put multi bandwidth. Use device API device is specified with -a", "overhead", 32},
+
+    {"ucp_put_multi_lat", UCX_PERF_API_UCP, UCX_PERF_CMD_PUT_MULTI, UCX_PERF_TEST_TYPE_PINGPONG,
+     "put multi latency. Use device API if device is specified with -a", "latency", 32},
```

Comment:
1. pls fix indent
2. the sentence sounds weird "put multi latency. Use device API if device is specified with -a"